scss/templates/user.scss

```diff
@@ -198,3 +181,105 @@
     }
   }
 }
+
+// User Navigation
+.user-secondary-navigation {
+  .new-private-message {
+    @include user-profile-btn;
+    width: 9rem;
+    margin-bottom: 1rem;
+  }
+
+  .nav-stacked {
+    li {
+      margin-bottom: 1.4375rem;
+
+      a {
+        color: $gray-100;
+        font-family: Libre Franklin;
```

Comment:
Why by default it doesn't pull this font-family?
_It is super odd that we have the need to declare font family in the middle of the CSS_

Reply:
Yes, this was my mistake. I'll remove it